docker/hbase/Dockerfile

```diff
@@ -20,15 +20,17 @@ ENV PACKAGE hbase-1.2.4
 
 WORKDIR /tmp/
 
-RUN    wget -q https://archive.apache.org/dist/hbase/1.2.4/${PACKAGE}-bin.tar.gz \
+RUN  wget -q https://archive.apache.org/dist/hbase/1.2.4/${PACKAGE}-bin.tar.gz \
```

Comment:
Here and in other places: you can download multiple files with wget like this: https://unix.stackexchange.com/a/669048